### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by kristianfjelde** [capacitor-community/google-maps#151] [General]
Hi!

First of all, thank you very much for this Plugin. I am currently working on a migration from Cordova to Capacitor, and since the Google Maps plugin is not compatible with Capacitor this plugin proves the most worthy alternative.

However, as I can see, the v2 of the plugins lacks some of the functionality that was available in the 1.2.0 version. Is there any roadmap available for upcoming functionality?

The following functionality is what I specifically request/am in need of:

- Set icon on markers (SVG or image url `data:image` preferable)
- Polylines and polygons (perhaps solved in this [PR](https://github.com/capacitor-community/google-maps/pull/149)?)
- Visible boundaries (sw, ne)

Thanks in advance!

**Comment by AE1NS** [comment]
I am also really excited to use this plugin in the future and its great to see what the community has built so far!

This are some references I am focused on:
- [Custom marker icon (Android)](https://github.com/capacitor-community/google-maps/issues/44#issuecomment-1069250133) "Android is still WIP and PRs are welcome"
- [Polyline, polygon and circle](https://github.com/capacitor-community/google-maps/issues/157)
- [Marker clustering](https://github.com/capacitor-community/google-maps/issues/80)
- [Dynamically change bounding rect](https://github.com/capacitor-community/google-maps/issues/156)
